app/invocation/dense/dense_invocation_overview.tsx

```diff
@@ -29,10 +28,10 @@ export default class DenseInvocationOverviewComponent extends React.Component<Pr
               {isBazelInvocation ? <>Invocation</> : <>Workflow invocation</>}
             </div>
             <div className="dense-invocation-invocation-id">
-              {this.props.invocationId} ({this.props.model.getFormattedStartedDate()})
+              {this.props.model.getId()} ({this.props.model.getFormattedStartedDate()})
```

Comment:
nit: we might want to call this `getInvocationId()` - I noticed we have a convention of being more verbose about having a qualifier before things named "id," presumably to avoid confusion

Reply:
Sure, why not